### ISSUE TITLE
Feature Request: Make compressing level configurable

### ISSUE THREAD

**Issue by claasd** [conan-io/conan#931]
Hi, as stated in #928, we have performance problems. 
We are compressing about 600MB of DLLs and PDBs. This takes up to several minutes. I manually changed the compression level, result in huge speed improvements and only marginally larger packages.

Suggestion: Change the method `gzopen_without_timestamps`:
```python
def gzopen_without_timestamps(name, mode="r", fileobj=None, compresslevel=None, **kwargs):
    """ !! Method overrided by laso to pass mtime=0 (!=None) to avoid time.time() was
        setted in Gzip file causing md5 to change. Not possible using the
        previous tarfile open because arguments are not passed to GzipFile constructor
    """
    from tarfile import CompressionError, ReadError
    if compresslevel == None:
         compresslevel = int(os.getenv("CONAN_COMRESSION_LEVEL", 9))
```

The default is maintained, but it can be overridden for speed improvements.

**Comment by lasote** [comment]
For the 0.20 release, we want to manage all these configurations also from the **conan.conf** but it's ok from now.

**Comment by claasd** [comment]
That was fast  thanks ;-)

**Comment by memsharded** [comment]
Released in 0.19!
